src/wallet/wallet.ts

```diff
@@ -1225,6 +1226,53 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return this.handleSendPreparedTransaction(tx);
   }
 
+  /**
+   * Call get token details API
+   *
+   * @param tokenId Token uid to get the token details
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  async getTokenDetails(tokenId: string) {
+    const response = await walletApi.getTokenDetails(this, tokenId);
+    const details: TokenDetailsObject = response.details;
+
+    // We should format the response to the same format the thin wallet currently responds
```

Comment:
Or the opposite. We designed the wallet service to have the best responses format to serve our wallets, then we've been formatting the old facade returns.

Reply:
Refactored on aaa03b768fd1c2dfa544fe73f32e58e2512bc3fe